src/types.ts

```diff
@@ -62,3 +71,33 @@ export type ChainID = number;
 export enum ColonyActionType {
   MintTokens = 'MINT_TOKENS',
 }
+
+export enum ColonyMotionType {
+  MintTokensMotion = 'MINT_TOKENS_MOTION',
+}
+/*
+ * Contract calls
+ */
+export enum ColonyOperations {
```

Comment:
A "ColonyOperation" being the underlying contract call, as distinct from a "ColonyAction" which may have multiple operations (e.g. the create colony flow)